### DISCUSSION TITLE
Be able to move objects between domains

### DISCUSSION THREAD

**Discussion by ab-smith** [intuitem/ciso-assistant-community#1751] [Ideas]
Be able to move objects between domains.
We didn't provide this feature natively, given the impact on the RBA, isolation, and relationships between objects.
We will work on a way of moving data between domains, but with some safeguards and extra checks to avoid any destructive operations.

**Comment by AndrzejRPiotrowski** [comment]
+1

**Comment by ciso62** [comment]
Hello, is there any roadmap regarding this feature ?
Due to some misunderstanding when we started using ciso-assistant, we did not create domains the way we should have and we would really like to correct our mistakes, so this would be very useful.
Thanks

**Comment by ab-smith** [comment]
it has been included a while back, plus the batch mode now helps with this.
